modules/nodes-edge-remote-workers-strategies.adoc

```diff
@@ -0,0 +1,157 @@
+// Module included in the following assemblies:
+//
+// * logging/nodes-edge-remote-workers.adoc
+
+[id="nodes-edge-remote-workers-strategies_{context}"]
+= About remote worker node strategies
+
+If you are using remote worker nodes, you should consider which objects to use to run your applications. 
+
+We recommend using DaemonSets or static pods based on the behavior you want in the event of network issues or power loss. In addition, you can use Kubernetes zones and tolerations to control or avoid pod evictions if the control plane cannot reach remote worker nodes.
```

Comment:
It is recommended to use DaemonSets or static pods based on the behavior you want in the event of network issues or power loss. In addition, you can use Kubernetes zones and tolerations to control or avoid pod evictions if the control plane cannot reach remote worker nodes.
```